cmd/sql/flow.go

```diff
@@ -451,6 +593,39 @@ func runCommand() *cobra.Command {
 	return cmd
 }
 
+type promptContent struct {
+	errorMsg string
+	label    string
+}
+
+func promptGetConfirmation(pc promptContent) (string, error) {
+	prompt := promptui.Select{
+		Label: pc.label,
+		Items: []string{"yes", "no"},
+	}
+
+	_, result, err := prompt.Run()
+	if err != nil {
+		fmt.Printf("Prompt failed %v\n", pc.errorMsg)
+		return "", err
+	}
+
+	return result, nil
+}
+
+//nolint:dupl
+func deployCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          deployCmdName,
+		RunE:         executeDeployCmd,
+		SilenceUsage: true,
+	}
+	cmd.Flags().StringVar(&env, "env", "default", "")
+	cmd.Flags().StringVar(&workflowName, "workflow-name", "", "")
+	cmd.Flags().StringVar(&projectDir, "project-dir", ".", "")
```

Comment:
please check once the output for `astro flow deploy --help`, I don't think so flow library which handles help output for other commands will work for this one, in which case we would need the default help out with description of these flags being set

Reply:
Thank you for the heads-up. This https://github.com/astronomer/astro-sdk/pull/1615 should handle that :)